src/main/java/com/kruize/environment/docker/DockerEnvImpl.java

```diff
@@ -16,21 +16,25 @@
 
 package com.kruize.environment.docker;
 
+import com.google.gson.JsonArray;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonParser;
 import com.kruize.analysis.docker.DockerAnalysisImpl;
 import com.kruize.environment.EnvTypeImpl;
 import com.kruize.metrics.ContainerMetrics;
 import com.kruize.query.PrometheusQuery;
 import com.kruize.recommendations.application.DockerApplicationRecommendations;
-import com.google.gson.JsonArray;
-import com.google.gson.JsonElement;
-import com.google.gson.JsonParser;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 import java.io.FileReader;
 import java.io.IOException;
 import java.util.ArrayList;
 
 public class DockerEnvImpl extends EnvTypeImpl
 {
+    private static Logger logger = LoggerFactory.getLogger(DockerEnvImpl.class);
```

Comment:
logger variable name can be consistent and as per coding guidelines

Reply:
Made all loggers final, and variable name to uppercase